modules.d/35network-wicked/module-setup.sh

```diff
@@ -42,35 +72,123 @@ install() {
+
+    # Enable systemd type units.
+    # TODO: do we need the wicked.service?
+    local i
+    for i in \
+        wicked.service \
+        wickedd.service \
+        wickedd-auto4.service \
+        wickedd-dhcp4.service \
+        wickedd-dhcp6.service \
+        wickedd-nanny.service \
+        wickedd-pppd@.service; do
+        $SYSTEMCTL -q --root "$initdir" enable "$i"
+    done
+
+    # Install the hosts local user configurations if enabled.
+    if [[ $hostonly ]]; then
+        inst_dir /etc/wicked/extensions
+
+        local wickeddbusconfdir
+        wickeddbusconfdir="$dbusconfdir"
+        if [ ! -f "$wickeddbusconfdir/org.opensuse.Network.conf" ]; then
+            if [ -f /etc/dbus-1/system.d/org.opensuse.Network.conf ]; then
+                wickeddbusconfdir="/etc/dbus-1/system.d"
+            elif [ -f /usr/share/dbus-1/system.d/org.opensuse.Network.conf ]; then
+                wickeddbusconfdir="/usr/share/dbus-1/system.d"
+            fi
+        fi
+
+        # TODO: do we need the wicked.service?
+        inst_multiple -H -o \
+            "/etc/wicked/*.xml" \
+            "/etc/wicked/extensions/*" \
+            "$wickeddbusconfdir"/org.opensuse.Network.conf \
+            "$wickeddbusconfdir"/org.opensuse.Network.AUTO4.conf \
+            "$wickeddbusconfdir"/org.opensuse.Network.DHCP4.conf \
+            "$wickeddbusconfdir"/org.opensuse.Network.DHCP6.conf \
+            "$wickeddbusconfdir"/org.opensuse.Network.Nanny.conf \
+            "$systemdsystemconfdir"/wicked.service \
+            "$systemdsystemconfdir/wicked.service/*.conf" \
+            "$systemdsystemconfdir/wicked@.service" \
+            "$systemdsystemconfdir/wicked@.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd.service \
+            "$systemdsystemconfdir/wickedd.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-auto4.service \
+            "$systemdsystemconfdir/wickedd-auto4.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-dhcp4.service \
+            "$systemdsystemconfdir/wickedd-dhcp4.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-dhcp6.service \
+            "$systemdsystemconfdir/wickedd-dhcp6.service/*.conf" \
+            "$systemdsystemconfdir"/wickedd-nanny.service \
+            "$systemdsystemconfdir/wickedd-nanny.service/*.conf" \
+            "$systemdsystemconfdir/wickedd-pppd@.service" \
+            "$systemdsystemconfdir/wickedd-pppd@.service.d/*.conf" \
+            /etc/libnl/classid
+
+        if [ -f /etc/dbus-1/system.d/org.opensuse.Network.conf ]; then
+            inst_multiple "/etc/dbus-1/system.d/org.opensuse.Network*"
+        elif [ -f /usr/share/dbus-1/system.d/org.opensuse.Network.conf ]; then
+            inst_multiple "/usr/share/dbus-1/system.d/org.opensuse.Network*"
+        fi
+
+        # SUSE specific configuration.
+        # TODO: does it make sense to copy the ifcfg files from running system?
```

Comment:
In order to apply static interface configurations from the installed system, this would be required. It is currently done in `network-legacy` as well.